code/modules/admin/admin_verbs.dm

```diff
@@ -637,3 +638,17 @@ var/list/admin_verbs_hideable = list(
 
 							testing("Spawned test mob with name \"[mob.name]\" at [tile.x],[tile.y],[tile.z]")
 			while (!area && --j > 0)
+
+/client/proc/toggle_AI_interact()
+	set name = "Toggle Admin AI Interact"
+	set category = "Admin"
+	set desc = "Allows you to interact with most machines as an AI would as a ghost"
+
+	if(!src.AI_Interact)

```

Comment:
This can be condensed into
`AI_Interact = !AI_Interact`
and
`has [AI_Interact ? "" : "de"] their`